### DISCUSSION TITLE
It would be very grateful if you could add a preview of the .caj file（如果能增加预览 .caj文档就好了）

### DISCUSSION THREAD

**Discussion by colapy93** [QL-Win/QuickLook#1478] [Q&A]
I don't know what's going on, but it seems that only a small part of .caj files can be previewed at present. But if there is a plug-in specifically for previewing .caj files, it would be very grateful.（目前好像只能打开很少一部分的.caj文档，不知道什么情况，但是如果有专门针对预览.caj文档的插件就会更好。）

**Comment by emako** [comment]
QL has never supported .caj files.

You need to develop your own plugin to support it.
